src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -4443,8 +4463,7 @@ class StubGenerator: public StubCodeGenerator {
       __ vadd_vv(vState1, vHash1, vState1);
 
       if (multi_block) {
-        int total_adds = vset_sew == Assembler::e32 ? 240 : 608;
-        __ addi(consts, consts, -total_adds);
+        __ subi(consts, consts, vset_sew == Assembler::e32 ? 240 : 608);
```

Comment:
makes the code less readable, maybe keep total_adds but keep the change of addi to subi ?

Reply:
Sure. Fixed.